.github/workflows/ci.yml

```diff
@@ -60,6 +90,8 @@ jobs:
         COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       name: Submit to coveralls
       run: coveralls
+    - name: Check coverage
+      run: coverage report -m --fail-under=100
```

Comment:
Interesting choice to fail below 100%.
I am afraid setting such a value may cause a sense of metric pleasing.

Is this percentage of coverage established at praw-dev

Reply:
It is.